src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/subheadline/index.js

```diff
@@ -2,10 +2,15 @@ import convertParagraph from '../paragraph';
 import { blockBase } from '#app/models/blocks';
 
 const convertToSubheadline = async block => {
-  const innerParagraph = await convertParagraph(block);
-  return blockBase('subheadline', {
-    blocks: [innerParagraph],
-  });
+  const typesArray = ['crosshead', 'heading', 'subheading'];
```

Comment:
if we had a case to convert some other block types n future, I can see this list getting longer. Is there a way we could better maintain this than coupling it to the scope of this function ?

Reply:
As on my other comment I think the idea of these changes is to be explicit about what CPS blocks the `subheadline` converter supports. I'm not sure we'd expect many other blocks to be supported and if we did, I'd be inclined to remove this guard and put it back to how it was, and just let it support any block.